docs/faq.md

```diff
@@ -66,4 +60,52 @@ TBD
 ### Is Luigi already being used within any products, or is it still too new?
 
 Yes, it is already being used in production and close-to-production within SAP. For example in Kyma, SAP C/4HANA Cockpit, Context Driven Services, Konduit and Varkes. Outside of SAP, SAAS AG (partner) uses Luigi. Additionally, there are some POCs going on and we're supporting a few other customers and partners who want to start using Luigi soon.
+
+
+### Can I authenticate Luigi with Google Cloud Identity?
+
+Yes, to use Luigi with a google account, follow these steps:
+
+1. Register a project and generate an OAuth2 Web Client based on [Google Developers Identity - OAuth2UserAgent](https://developers.google.com/identity/protocols/OAuth2UserAgent).
+2. To get your app running locally, set the Autorized JavaScript Origins URIs to `http://localhost:4200` and Authorized redirect URIs to `http://localhost:4200/luigi-core/auth/oauth2/callback.html?storageType=localStorage`.
+3. Copy the Client ID on the right side, ending with `apps.googleusercontent.com`
+4. Update the LuigiConfig auth section. We have added also the parts for logout and getting user information.
+   ```javascript
```

Comment:
```suggestion
```javascript
```

It looks like this section is not rendered correctly inside a code block 🤔 I'm not sure why, but it could be the space before ```
https://github.com/SAP/luigi/blob/fe2b144a9251275058d2d778e3dad88d1fea3efa/docs/faq.md